manager/volume.go

```diff
@@ -263,9 +263,7 @@ func (m *VolumeManager) Attach(name, nodeID string, disableFrontend bool, attach
 	return v, nil
 }
 
-// Detach will handle regular detachment as well as volume migration confirmation/rollback
-// if nodeID is not specified, the volume will be detached from all nodes
-func (m *VolumeManager) Detach(name, attachmentID string, forceDetach bool) (v *longhorn.Volume, err error) {
+func (m *VolumeManager) Detach(name, attachmentID, hostID string, forceDetach bool) (v *longhorn.Volume, err error) {
```

Comment:
In general, please remember to add a doc comment for any exported interface/method/function.

Reply:
Thank you. I added the correct comment